content/docs/using-pulumi/crossguard/snyk-policy.md

```diff
@@ -0,0 +1,256 @@
+---
+title_tag: "Snyk Container Scanning | CrossGuard"
+meta_desc: Snyk container scanning helps you ensure that your Docker images are secure.
+title: Synk Container Scanning
+h1: Snyk Container Scanning
+weight: 4
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+  usingpulumi:
+    parent: crossguard
+aliases:
+- /docs/guides/crossguard/snyk-container-scanning/
+---
+<!-- markdownlint-disable ul code -->
+
+## Overview
+
+[Snyk container scanning](https://github.com/pulumi/templates-policy/tree/master/snyk-typescript) is a Pulumi policy as code template that allows you to uses the [Snyk CLI](https://docs.snyk.io/scan-using-snyk/snyk-container/scan-container-images) to scan [`docker.Image`](/registry/packages/docker/api-docs/image/) resources for vulnerabilities.
+
+For more information on Pulumi's Policy as Code solution, see [Get started with Pulumi policy as code](docs/using-pulumi/crossguard/get-started/).
+
+## Using Snyk Container Scanning
+
+In this guide, we'll show you how to create a Policy Pack that enables Snyk container scanning.
+
+### Prerequisites
+
+- [Install Pulumi](/docs/install/)
+- [Install Node.js](https://nodejs.org/en/download/)
+- [Install the Snyk CLI](https://docs.snyk.io/snyk-cli)
+
+### Running the policy pack
+
+This guide assumes the following directory structure:
+
+```text
+.
+├── infra # Your Pulumi IaC program lives here.
+└── policy # Your policy pack will live here. Directory is assumed to be initially empty.
+```
+
+Your Pulumi IaC program should have at least one `docker.Image` resource, similar to the following:
+
+{{< chooser language "typescript,python,go,csharp,java,yaml" / >}}
```

Comment:
Have you verified all of these examples work? (Just asking because I haven't.)

Reply:
I did YAML -> `pulumi convert` and TypeScript works, so I think that's sufficient. (Let me know if you feel differently.)

Reply:
In general I do test all code samples, so if it's feasible to do that, I'd recommend it.